catalog/plugin/catalog-plugin-metacard-validation/src/main/java/ddf/catalog/metacard/validation/MetacardValidityMarkerPlugin.java

```diff
@@ -115,6 +121,15 @@ public UpdateRequest process(UpdateRequest input)
                     getValidationProblems(validatorName, e, errors, warnings, counter);
                 }
             }
+
+            List<Serializable> tags = new ArrayList<>();
```

Comment:
You should be able to construct the list with `metacard.getTags()` and collapse these two lines to one.